inst/www/shiny-semantic-dropdown.js

```diff
@@ -51,33 +45,30 @@ $.extend(semanticDropdownBinding, {
   // callback is a function that queues data to be sent to
   // the server.
   subscribe: function(el, callback) {
-    $(el).on('keyup change', function () { callback(); });
+    $(el).dropdown({
+      onChange: function(value, text, $selectedItem) {
+        callback();
+      }
+    })
```

Comment:
I think this could be shortened, but not sure:
```
subscribe: function(el, callback) {
    $(el).dropdown({
      onChange: callback
    });
  },
```
tested in dropdown_updates app and it worked

Reply:
thanks, done